### PR TITLE
HA-WyzeAPI 2021.11.1

### PR DESCRIPTION
**Pull request by JoshuaMulliken**
# Change Log

## Features

* Battery Sensors for Lock/Lock Keypad and Wyze Cam Outdoor
* Camera Events via Event Bus

## Refactoring

* Store UUID in Config Entry
* init.py cleanup

### CODE REVIEW

**Comment by JoshuaMulliken** [comment]
@yoinx I want to make sure nothing gets messed up in the merge process

**Comment by yoinx** [comment]
> @yoinx I want to make sure nothing gets messed up in the merge process

It looks like the first 4 commits in this list would be things that were already in master. Not sure what's going on there. Everything after that *looks* like the right commits.

File: custom_components/wyzeapi/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "domain": "wyzeapi",
-  "version": "2021.10.2",
+  "version": "2021.11.1b1",
```
Comment (yoinx) [inline custom_components/wyzeapi/manifest.json:3]: Should this be b1? The PR title is 2021.11.1

**Review by yoinx — CHANGES_REQUESTED** [review]
Looks good to me other than the "2021.11.b1" version of the title of the PR is for a stable release

File: custom_components/wyzeapi/manifest.json (hunk 1 above)
Comment (JoshuaMulliken) [reply custom_components/wyzeapi/manifest.json:3]: Ahh true. I should fix that

**Pull request merged**